lib/matplotlib/testing/conversion_cache.py

```diff
@@ -0,0 +1,200 @@
+"""
```

Comment:
I think we need to make this module private, as well as all the classes in it. We shouldn't expose this API (at least just yet) to users.